### ISSUE TITLE
labels with .PNG are not considered .png. Causes TypeError

### ISSUE THREAD

**Issue by RoshanStacker** [Labelbox/labelbox-python#740]
Uploading photo from an Iphone (for example) has the filename `IMG_6371.PNG` for example. 

`labelbox/data/serialization/labelbox_v1/label.py` only checks for lowercase file extensions. 

This causes a TypeError:
```
/usr/local/lib/python3.7/dist-packages/labelbox/data/serialization/labelbox_v1/label.py in _infer_media_type(self)
    226                 #  We will eventually provide the media type in the export.
    227                 raise TypeError(
--> 228                     f"Can't infer data type from row data. row_data: {self.row_data[:200]}"
    229                 )
    230
```

**Comment by msokoloff1** [comment]
Will be fixed by: https://github.com/Labelbox/labelbox-python/pull/752/files

**Comment by msokoloff1** [comment]
This was fixed in 3.29.0. Feel free to open if you are still having any issues.
